Added corrections to android getting started guide (#669)
MapsPeople/docs: src/android/v3/getting-started/new-project.md

This guide explains how to start using a MapsIndoors map in your Android application using the MapsIndoors Android SDK v3.

We recommend using Android Studio for using this tutorial. Read how to set it up here: [Installing Android Studio](https://developer.android.com/studio/install)

> If you do not have a Android device, you can setup an emulator through Android studio, you can see how to do this here: [Setting up Android emulator](https://developer.android.com/studio/run/emulator).

Reviewer: ```suggestion
If you do not have a Android device, you can setup an emulator through Android studio. You can see how to do that here: [Setting up Android emulator](https://developer.android.com/studio/run/emulator).
```

Reviewer 2: ```suggestion
If you do not have an Android device, you can setup an emulator through Android studio, you can see how to do this here: [Setting up Android emulator](https://developer.android.com/studio/run/emulator).
```

> If you do not have a Android device, you can setup an emulator through Android studio. You can see how to do that here: [Setting up Android emulator](https://developer.android.com/studio/run/emulator).

Reviewer 3: ```suggestion
If you do not have a Android device, you can setup an emulator through Android Studio. You can see how to do that here: [Setting up Android emulator](https://developer.android.com/studio/run/emulator).
```

> If you already have a Android device make sure to enable developer mode and USB debugging. Read about this here [Enable USB debugging](https://developer.android.com/studio/debug/dev-options#enable)

Reviewer: ```suggestion
If you already have an Android device, make sure to enable developer mode and USB debugging. Read about this here: [Enable USB debugging](https://developer.android.com/studio/debug/dev-options#enable)
```

To benefit from the guides, you will need basic knowledge about:

* Android Development
* Google Maps Android API

You can get started in two ways, either by reviewing and modifying the [basic example](#basic-example) or do the [clean setup](#setup-mapsindoors). We recommend following the basic example.

## Basic Example

The tutorial will be based on you starting from our basic map implementation. This contains basic UI implementations together with layout files and drawables used to create the UI. You will then be guided through how to implement the MapsIndoors SDK into this app.

> The basic example contains a single `activity` app with already made `fragments` to host the different logic to get a complete app interracting with a map and `MapsIndoors` data.

Reviewer: ```suggestion
The basic example contains a single `activity` app with already made `fragments` to host the different logic to get a complete app interacting with a map and `MapsIndoors` data.
```

Add the following dependencies and the MapsIndoors maven repository:

`Gson` and `okhttp` is used by MapsIndoors to function properly with network calls and deserializing.

> `play-services-maps` is used for the Google maps which MapsIndoors is build ontop of on Android.

Reviewer: ```suggestion
`play-services-maps` is used for Google Maps which MapsIndoors is build on top of on Android.
```
